lua/gitsigns/git/repo.lua

```diff
@@ -329,7 +331,7 @@ function M:ls_files(file)
     '--others',
     '--exclude-standard',
     has_eol and '--eol',
-    file,
+    self:relpath(file),
```

Comment:
This won't work if `file` is a link.

```suggestion
    util.normalize_path(file),
```

Reply:
> This won't work if `file` is a link.

Why this will not work if this is a link? Give me an example. Absolute path, leads outside repo also will not work.

We need to create relative path, but `normalize_path` - will create absolute path.

Let's decide again base things.
Do you agree, that it should use relative paths?
- Pro - relative paths will work regardless of in what format are work-tree and git-dir are specifies, and under which OS git is running.
- Cons - I have no arguments against.

Reply:
> Absolute path, leads outside repo also will not work.

Absolute path, lead inside repo also will not work.